.github/workflows/cri_minio_test.yml

```
@@ -42,26 +43,18 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
-    - name: Setup firecracker-containerd
-      run: ./scripts/setup_firecracker_containerd.sh
-
     - name: Build
       run: go build
 
-    - name: Start vHive cluster
-      run: ./scripts/cloudlab/start_onenode_vhive_cluster.sh
-
     - name: modify $PATH
       run: echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
 
     - name: Setup minio
       run: sleep 1m && make -C ./function-images/tests/save_load_minio local
```

```suggestion
      run: sleep 1m && make -C ./function-images/tests/save_load_minio build_local
```

there is a [protoc not found error](https://github.com/ease-lab/vhive/pull/334/checks?check_run_id=3123380919#step:7:27) in the CI. let's fix it by skipping the proto generation in the CI
```suggestion
      run: sleep 1m && make -C ./function-images/tests/save_load_minio local
```